deepchecks/vision/checks/model_evaluation/mean_average_precision_report.py

```diff
@@ -104,14 +104,21 @@ def add_condition_mean_average_precision_not_less_than(self: MPR, min_score: flo
         """
         def condition(df: pd.DataFrame):
             not_passed = defaultdict(dict)
+            min_score_info = np.inf, None, None
             for index, column in zip(df.index, df.columns):
-                if df.loc[index, column] < min_score:
-                    not_passed[index][column] = format_number(df.loc[index, column], 3)
+                score = df.loc[index, column]
+                if score < min_score:
+                    not_passed[index][column] = format_number(score, 3)
+                # Tracking the smallest score to print if condition passed
+                if score < min_score_info[0]:
+                    min_score_info = score, index, column
             if len(not_passed):
-                details = f'Found scores below threshold:\n' \
-                          f'{dict(not_passed)}'
+                details = f'Found scores below threshold:\n{dict(not_passed)}'
```

Comment:
maybe only print those with lowest threshold
same to what you did with top values in dict